woocommerce/woocommerce-android
# Stripe SDK update. Merge back

### Description

The PR contains changes related to the usage of the new SDK version. All code was tested and reviewed

### Testing instructions

Just a sanity check is needed. @AnirudhBhat @malinajirka  let's just go through the code and verify that the changes are all relevant to the SDK update.

@malinajirka could you please briefly check if it works with M2 as expected too?
